http/user-authorizations-tests.http

```diff
@@ -57,6 +57,61 @@ Accept: application/json
     client.assert(response.status === 201, "Response status is not 201");
     client.global.set("USERNAME", response.body.username);
   });
+
+  client.test("Cache Control headers should be present", function () {
+    client.assert(response.headers.valueOf("Cache-Control"), "Cache-Control header is missing");
+    client.assert(response.headers.valueOf("Pragma"), "Pragma header is missing");
+    client.assert(response.headers.valueOf("Expires"), "Expires header is missing");
+  });
+
+  client.test("Content Security Policy header should be present", function () {
+    client.assert(response.headers.valueOf("Content-Security-Policy"), "Content-Security-Policy header is missing");
+  });
+
+  client.test("X-Content-Type-Options header should be present", function () {
+    const contentTypeOptions = response.headers.valueOf("X-Content-Type-Options");
+    client.assert(contentTypeOptions, "X-Content-Type-Options header is missing");
+    client.assert(contentTypeOptions === "nosniff", "X-Content-Type-Options should be 'nosniff'");
+  });
+
+  client.test("Cross-Origin-Embedder-Policy header should be present", function () {
+    const coep = response.headers.valueOf("Cross-Origin-Embedder-Policy");
+    client.assert(coep, "Cross-Origin-Embedder-Policy header is missing");
+    client.assert(coep === "unsafe-none" || coep === "require-corp", "Cross-Origin-Embedder-Policy should be 'unsafe-none' or 'require-corp'");
```

Comment:
@sergiizakharov why do this test (and the others below) test for multiple possible values instead of the default value? (based on https://docs.camunda.io/docs/next/self-managed/components/orchestration-cluster/identity/configuration/#http-security-headers)

Reply:
Thanks!